PiwikTracker/Device.swift

```diff
@@ -1,9 +1,9 @@
 import UIKit
 
-internal struct Device {
+public struct Device {
```

Comment:
I agree that the users should select by themselves which dimensions they want to set unless that doesn't conflict with the piwik API.

I am also not sure about making Device and Application public. It is no feature one would expect from a tracker but I son't see anything serious to be said against it.